source/usage-examples/updateOne.txt

```diff
@@ -3,3 +3,68 @@ Update a Document
 =================
 
 .. default-domain:: mongodb
+
+You can update a document in a collection by using the ``Collection.UpdateOne()``
+method. 
+
+This example passes a query filter and an update parameter to the
+``UpdateOne()`` method. The query filter specifies the document to
+match, while the update parameter specifies the change to
+make to the document. The following code example calls the
+``UpdateOne()`` method on a document matched by its ``_id`` field from
+the ``restaurants`` collection to create a new field called
+``avg_rating`` with a value of 4.4:
```

Comment:
nit/suggestion: Since we're going for streamlined and simple, I'd put slightly less emphasis on the query filter. Additionally, I don't think it's necessary to say `the following code example`, just `the following example`.

```suggestion
The following example passes a query filter and an update parameter to the
``UpdateOne()`` method, matching a document in the ``restaurants`` collection
by the value of its ``_id`` field and creating a new field in that document called
``avg_rating`` with a value of *4.4*:
```

Reply:
accepted